screens/Post.tsx

```diff
@@ -1,21 +1,81 @@
-import { StatusBar } from "expo-status-bar";
-import { StyleSheet, Text, View } from "react-native";
-import { COLORS } from "../theme";
+import { useState, React } from "react";
+import { TextInput, TouchableOpacity, View, Text } from "react-native";
+import { SHADOWS, COLORS } from "../theme";
+import { baseUrl } from "../env";
+import axios from "axios";
+
+const Post = ({ navigation }) => {
+  const [postText, changePostText] = useState("");
+  const [postError, setPostError] = useState("");
+
+  const createPost = () => {
+    axios
+      .post(`${baseUrl}posts`, {
+        // Daten
+      })
+      .then(function (response) {
+        if (response.status === 201) {
+          setPostError("");
+          navigation.navigate("TopBar");
+        }
+      })
+      .catch(function (error) {
+        switch (error.response.status) {
+          case 400: {
+            setPostError(
+              "The request body is invalid. Please check the request body and try again.",
+            );
+            break;
+          }
+          case 401: {
+            setPostError("Unauthorized");
+            break;
+          }
+        }
+      });
+  };
 
-export default function Post() {
   return (
-    <View style={styles.container}>
-      <Text>Post</Text>
-      <StatusBar style="auto" />
+    <View className="bg-white flex-1">
+      <View className="bg-white justify-center flex-row">
+        <TextInput
+          className="flex-1 border-2 mt-10 ml-2.5 mr-2.5 border-[#ccc] rounded-[8px] p-2"
+          value={postText}
+          onChangeText={(post) => {
+            changePostText(post);
+          }}
+          multiline={true}
+          numberOfLines={8} // Anzahl der sichtbaren Zeilen
+          placeholder="Verfasse hier deinen Text ..."
+          maxLength={256}
+        />
+      </View>
+      <View className="mt-1.5 ml-2.5 justify-start flex-row">
+        <Text className="text-black text-xs">{postText.length} / 256</Text>
+      </View>
+      <View>
+        {!(postError.length === 0) && (
```

Comment:
Warum frägst du hier auf die länge des postErrors ab? Kann man das nicht auch mit null oder ähnlichem machen? Oder muss das in React so abgefragt werden? Ich weiß es nicht aber sieht iwie off aus

Reply:
Passt so. War nur unwissend :)